packages/server/src/main/java/software/uncharted/terarium/hmiserver/proxies/funman/FunmanProxy.java

```diff
@@ -0,0 +1,25 @@
+package software.uncharted.terarium.hmiserver.proxies.funman;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import com.fasterxml.jackson.databind.JsonNode;
+
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import software.uncharted.terarium.hmiserver.proxies.funman.FunmanProxy;
+
+@FeignClient(name = "funman-api", url = "${funman-service.url}")
```

Comment:
In the end its all the same, but for the url path we had been following the convention where in application.properties you would have `http://funman.staging.terarium.ai/api` and here in the FeignClient the path would be `/queries